### ISSUE TITLE
Create factories and tests for the entities app in the backend 

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#508]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The task is to create test data (factories) with [factory_boy](https://factoryboy.readthedocs.io/en/stable/) for the models in the entities app. 

```bash
Name                            Stmts   Miss  Cover
---------------------------------------------------
...
entities\models.py                100     13    87%
...
```
Have a look at the `content/factory.py` and `content/tests.py` to get started.

You can run `pytest` inside backend directory and a report terminal and html report will be generated. The html report shows you excatly what needs to be tested. The html report will saved in the folder "htmlcov".

### Contribution

Happy to help someone with this task.

**Comment by marcosquadross** [comment]
Hey, I would like to work on this issue. Can I try to do this issue?

**Comment by andrewtavis** [comment]
Hey @marcosquadross! Thanks for your willingness to help our :) :) Let us know if anything comes that you need help with!

**Comment by marcosquadross** [comment]
I have a question about this comment in the ```entities/model.py```:

`TODO: Some relational-models may need to be moved in the "events" or "content" app in order to prevent circular dependency issues.`

What do I need to do?

**Comment by to-sta** [comment]
> I have a question about this comment in the `entities/model.py`:
> 
> `TODO: Some relational-models may need to be moved in the "events" or "content" app in order to prevent circular dependency issues.`
> 
> What do I need to do?

@marcosquadross you can ignore that comment.

I think this was still in there before we used strings to represent the related model inside a ForeignKey field.

@momanyisamuel I think we can get rid of this comment right?

**Comment by andrewtavis** [comment]
Closed via #538! Thank you, @marcosquadross! 🚀 

I removed the comments from the backend file headers in 1529a33. I think we're good without all of them, but if one's missing then let's open an issue 😊
